src/test/java/core/basesyntax/service/impl/FileServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileServiceImplTest {
+    public static final String PATH_TO_READ_FILE = "src/test/resources/input_file";
+    public static final String PATH_TO_WRITE_FILE = "src/test/resources/report_file";
+    public static final String WRONG_FILEPATH = "src/test/resources/wrong_file";
+    public static final String REPORT_DATA = "fruits, quantity, banana,152, apple, 90";
+    private FileService fileService;
+
+    @Before
+    public void setUp() {
+        fileService = new FileServiceImpl();
+    }
+
+    @Test
+    public void parseDataFromFile_Ok() {
+        List<String> expectedData = List.of("type,fruit,quantity", "b,banana,20", "b,apple,100",
+                "s,banana,100", "p,banana,13", "r,apple,10", "p,apple,20", "p,banana,5",
+                "s,banana,50");
+        List<String> actualData = fileService.parseDataFromFile(PATH_TO_READ_FILE);
+        assertEquals(expectedData, actualData);
+    }
+
+    @Test
+    public void parseDataFromFile_wrongPath_NotOk() {
+        try {
+            fileService.parseDataFromFile(WRONG_FILEPATH);
+        } catch (Exception e) {
+            throw new RuntimeException("wrong path from file to read: "
```

Comment:
make it like this:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/4c3d0dfb-7230-4ef6-afc4-aa5577894a2e)